pkg/configpb/configpb.pb.go

```diff
@@ -62,7 +62,7 @@ func (x StatusCode) String() string {
 	return proto.EnumName(StatusCode_name, int32(x))
 }
 func (StatusCode) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_configpb_0380a73eaf0b444c, []int{0}
```

Comment:
revert changes to these files

Reply:
Done, but how those files can be corrupted... 🤔

Reply:
well these random numbers are regenerated every time 🤔